src/io/dicom.ts

```diff
@@ -106,7 +109,20 @@ export class DICOMIO {
 
     const result = await this.runTask('dicom', args, inputs, outputs);
 
-    return JSON.parse((result.outputs[0].data as TextStream).data);
+    // File names are indexes into input files array
+    const volumeToFileIndexes = JSON.parse(
+      (result.outputs[0].data as TextStream).data
+    ) as VolumesToFileNamesMap;
+
+    const volumeToFiles = Object.fromEntries(
+      Object.entries(volumeToFileIndexes).map(([volumeKey, fileIndexes]) => [
+        volumeKey,
+        // unique file indexes to Files
+        fileIndexes.map((fileIndex) => files[parseInt(fileIndex, 10)]),
```

Comment:
Same here. `parseInt` defaults to base 10, so no need to specify it here.

Reply:
Kept radix here due to eslint rule

Reply:
Huh, so there is an eslint rule for that. I'm fine with keeping it then.